docs/manual/common/guide/devmode/CassandraServer.md

```diff
@@ -156,4 +156,25 @@ In sbt:
 
 @[local-instance](code/build-cassandra-opts3.sbt)
 
-Assuming your local Cassandra instance is running on port `9042`.
+These two settings will only be used when running Lagom in DevMode. The purpose of these two settings is to disabled the embedded Cassandra server and configure the Service Locator in DevMode to still be able to locate Cassandra when looking for `cas_native`.
```

Comment:
I'm wondering if it should say why you would disable the embedded Cassandra - typically, you would do this if you were already running your own Cassandra locally, perhaps you have another application not managed by Lagom that uses Cassandra, and for running that you run Cassandra, and it doesn't make sense for Lagom to start its own Cassandra then.